MANIFEST.in

```diff
@@ -1,3 +1,6 @@
 include LICENSE
 include README.md
```

Comment:
You can drop these inclusions because they are auto-included in modern setuptools + setuptools-scm includes everything that is checked in to Git.
```suggestion
```